services/squareone/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
I suspect this won't ever do anything given our current setup, although no objections to leaving it if you think it might be useful some day.